src/modules/commander/Commander.cpp

```diff
@@ -1237,6 +1237,31 @@ Commander::set_home_position_alt_only()
 	return false;
 }
 
+void
+Commander::updateHomePositionYaw(float yaw)
+{
+	home_position_s home = _home_pub.get();
+
+	home.yaw = yaw;
+	home.timestamp = hrt_absolute_time();
+
+	_home_pub.update(home);
+}
+
+void
+Commander::checkEkfResetCounters()
+{
+	if (_attitude_sub.get().quat_reset_counter != _quat_reset_counter) {
```

Comment:
Thinking about this again, unless we want to remove `yaw` from `vehicle_local_position` should we instead add a yaw reset counter and delta yaw?